bin/beeline

```diff
@@ -23,8 +23,10 @@
 # Enter posix mode for bash
 set -o posix
 
-# Figure out where Spark is installed
-FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
+# Figure out if SPARK_HOME is set
+if [ -z "${SPARK_HOME}" ]; then
+    export SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
```

Comment:
Yes, I very much like how this has standardized everything to use `SPARK_HOME` instead of `FWDIR`, and doesn't overwrite the value if already set. (Nit: all of the occurrences of this line have a 4-space indent instead of 2)

LGTM; does anyone see a reason this isn't a good idea? I suppose now `SPARK_HOME`, if set, has an effect everywhere, but it looks like the desired effect. Docs also make reference to `SPARK_HOME` as if it has this effect.